### PR TITLE
feat: redesign widget cart and wording

### PR DESCRIPTION
**Pull request by Benjamin-Freoua-Alma**
### Reason for change

[Linear task](https://linear.app/almapay/issue/ECOM-1814/[💅-ps]-redesign-widget)
[Linear task](https://linear.app/almapay/issue/ECOM-1812/[💅-ps]-improve-wording-on-widget-in-cart-from-ps)

### Code changes

Update wording button
Add bold for the product name

### How to test

_As a reviewer, you are encouraged to test the PR locally._

See the item insurance. in the cart

### Checklist for authors and reviewers

- [ ] The title of the PR uses business wording, not technical jargon, for the changelog readers to understand it
- [ ] The PR implements the changes asked in the referenced task / issue
- [ ] The automated tests are compliant with the [testing strategy](https://www.notion.so/almapay/Backend-testing-strategy-06c642cec1bf47b9b8feca3a91ea8d4a)
- [ ] The tests are relevant, and cover the corner/error cases, not only the happy path
- [ ] You understand the impact of this PR on existing code/features
- [ ] The changes include adequate logging and Datadog traces
- [ ] Documentation is updated (API, developer documentation, ADR, Notion...)

### CODE REVIEW

File: alma/alma.php [hunk 1]
```diff
@@ -151,12 +151,9 @@ public function handlePSModule()
      */
     public function checkCompatibilityPSModule()
     {
-        return false;
```
Comment (FranceBe) [inline alma/alma.php:154]: I feel like this PR includes changes that are not related to the Cart Item design fix ?

Comment (FranceBe) [reply alma/alma.php:154]: @Benjamin-Freoua-Alma  Should this PR be rebased or target a different branch for merging ? 🤔

Comment (Benjamin-Freoua-Alma) [reply alma/alma.php]: @FranceBe I rebased the main branch of cart-item and I forgot to rebase this too ^^
I fixed it

**Review by FranceBe — APPROVED** [review]
💪 
Can you send a screenshot to Michael for PrD validation as well ? :)

**Comment by Benjamin-Freoua-Alma** [comment]
> Member

@FranceBe Ok I'll send him that
Capture d’écran 2024-07-01 à 11 42 09

**Comment by Benjamin-Freoua-Alma** [comment]
@FranceBe Design validated by Michael for my widget
Capture d’écran 2024-07-01 à 13 28 59
